packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -17,12 +17,14 @@ import {
 
 const precedence = {
   silent: 0,
-  error: 1,
-  warn: 2,
-  log: 3,
-  info: 4,
-  debug: 5,
-  trace: 6
+  group: 1,
```

Comment:
Not sure what is going on here, is this needed as part of the PR? Could we make it a separate commit with `logger` being the scope?